dateutil/parser/_parser.py

```diff
@@ -571,6 +571,74 @@ def resolve_ymd(self, yearfirst, dayfirst):
         return year, month, day
 
 
+def next_jump(tokens, idx, skip_comma=True):
+    """
+    Get the next sibling and its index, skipping over whitespace
```

Comment:
An example of what this is looking for would probably be useful to future software archaeologists. I think you have one somewhere in the comments of the function itself.

Reply:
Good idea, will do.